test_scripts/API/SetAppIcon/002_Mobile_app_registration_with_param_iconResumed_true.lua

```diff
@@ -0,0 +1,55 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0041-appicon-resumption.md
+-- User story:TBD
+-- Use case:TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case:
+-- 1) SDL, HMI are started.
+-- 2) Mobile application is registered and sets custom icon via sending PutFile and valid SetAppIcon request.
+-- 3) Mobile app is re-registered.
+-- SDL does:
+-- 1) Successfully register application.
+-- 2) Successful process PutFile and SetAppIcon requests.
+-- 3) Respond with result code "SUCCESS" and "iconResumed" = true for RAI request.
+-- 4) Send to HMI OnAppRegistered notification "icon" and notify that apps icon exists.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/SetAppIcon/commonIconResumed')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local requestParams = {
```

Comment:
@HSavynetska It's better not to use temporary variables ```requestParams``` and ```requestUiParams``` and to define the values directly in ```allParams```